analytics/terraform/spark-k8s-operator/examples/karpenter/nvme-ephemeral-storage/nvme-ephemeral-storage.yaml

```diff
@@ -42,7 +42,7 @@ spec:
     "spark.app.name": "taxi-trip"
     "spark.kubernetes.driver.pod.name": "taxi-trip"
     "spark.kubernetes.executor.podNamePrefix": "taxi-trip"
-    "spark.local.dir": "/data1"
+    # "spark.local.dir": "/data1"
```

Comment:
Since the local.dir is `spark.local.dir": "/data` I think we can remove it from here